
time="2025-08-02T13:12:21-03:00" level=warning msg="/home/rinha-user/rinha-de-backend-2025/participantes/Maxel-Uds-quarkus/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion"
 Network maxel-uds-quarkus_payment-router  Creating
 Network maxel-uds-quarkus_payment-router  Created
 Container api2  Creating
 Container api1  Creating
 Container api1  Created
 Container api2  Created
 Container load_balancer  Creating
 Container load_balancer  Created
Attaching to api1, api2, load_balancer
api2           | __  ____  __  _____   ___  __ ____  ______ 
api2           |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api2           |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api2           | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api2           | 2025-08-02 16:12:23,191 INFO  [io.quarkus] (main) payment-storage 1.0.0 native (powered by Quarkus 3.25.0) started in 0.027s. Listening on: http://0.0.0.0:8080
api2           | 2025-08-02 16:12:23,191 INFO  [io.quarkus] (main) Profile prod activated. 
api2           | 2025-08-02 16:12:23,191 INFO  [io.quarkus] (main) Installed features: [cdi, rest, rest-jackson, smallrye-context-propagation, vertx]
api1           | __  ____  __  _____   ___  __ ____  ______ 
api1           |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api1           |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api1           | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api1           | 2025-08-02 16:12:23,224 INFO  [io.quarkus] (main) payment-router 10.0.0 native (powered by Quarkus 3.11.1) started in 0.034s. Listening on: http://0.0.0.0:8080
api1           | 2025-08-02 16:12:23,224 INFO  [io.quarkus] (main) Profile prod activated. 
api1           | 2025-08-02 16:12:23,224 INFO  [io.quarkus] (main) Installed features: [cdi, rest, rest-client, rest-client-jackson, rest-jackson, smallrye-context-propagation, vertx]
load_balancer  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
load_balancer  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
load_balancer  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
load_balancer  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
load_balancer  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
load_balancer  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
load_balancer  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
load_balancer  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
load_balancer  | /docker-entrypoint.sh: Configuration complete; ready for start up
load_balancer  | 2025/08/02 16:12:31 [error] 29#29: *3 open() "/etc/nginx/html/purge-payments" failed (2: No such file or directory), client: 172.20.0.1, server: , request: "POST /purge-payments HTTP/1.1", host: "localhost:9999"
[Kload_balancer exited with code 0
api2           | 2025-08-02 16:13:33,697 INFO  [io.quarkus] (Shutdown thread) payment-storage stopped in 0.005s
[Kapi2 exited with code 143
api1           | 2025-08-02 16:13:38,695 INFO  [io.qua.thread-pool] (Shutdown thread) Awaiting thread pool shutdown; 15 thread(s) running with 0 task(s) waiting
api1           | 2025-08-02 16:13:43,695 ERROR [org.jbo.thr.errors] (executor-thread-10) Thread Thread[#50,executor-thread-10,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,695 ERROR [org.jbo.thr.errors] (executor-thread-8) Thread Thread[#48,executor-thread-8,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,695 ERROR [org.jbo.thr.errors] (executor-thread-13) Thread Thread[#53,executor-thread-13,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,695 ERROR [org.jbo.thr.errors] (executor-thread-9) Thread Thread[#49,executor-thread-9,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,695 ERROR [org.jbo.thr.errors] (executor-thread-11) Thread Thread[#51,executor-thread-11,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,695 ERROR [org.jbo.thr.errors] (executor-thread-5) Thread Thread[#45,executor-thread-5,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,695 ERROR [org.jbo.thr.errors] (executor-thread-7) Thread Thread[#47,executor-thread-7,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,696 ERROR [org.jbo.thr.errors] (executor-thread-1) Thread Thread[#36,executor-thread-1,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,696 ERROR [org.jbo.thr.errors] (executor-thread-4) Thread Thread[#44,executor-thread-4,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,696 ERROR [org.jbo.thr.errors] (executor-thread-6) Thread Thread[#46,executor-thread-6,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,696 ERROR [org.jbo.thr.errors] (executor-thread-2) Thread Thread[#42,executor-thread-2,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,696 ERROR [org.jbo.thr.errors] (executor-thread-14) Thread Thread[#54,executor-thread-14,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,696 INFO  [io.qua.thread-pool] (Shutdown thread) Awaiting thread pool shutdown; 11 thread(s) running with 0 task(s) waiting
api1           | 2025-08-02 16:13:43,696 ERROR [org.jbo.thr.errors] (executor-thread-15) Thread Thread[#55,executor-thread-15,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,696 ERROR [org.jbo.thr.errors] (executor-thread-3) Thread Thread[#43,executor-thread-3,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,696 ERROR [org.jbo.thr.errors] (executor-thread-12) Thread Thread[#52,executor-thread-12,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api1           | 	... 13 more
api1           | 
api1           | 2025-08-02 16:13:43,697 INFO  [io.quarkus] (Shutdown thread) payment-router stopped in 10.006s
[Kapi1 exited with code 143
log truncated at line 1000
